strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -402,27 +429,61 @@ public void testRemoveTimestampedSnapshots(@MavenRepository(repositoryId = TRTS_
             assertEquals(1, timestampedSnapshots, "Amount of timestamped snapshots doesn't equal 1.");
 
             Path snapshotArtifactPath = getSnapshotArtifactPath(artifactVersionBasePath);
+            assertNotNull(snapshotArtifactPath);
             assertTrue(snapshotArtifactPath.toString().endsWith("-3.jar"));
+        }
 
+        // To check removing timestamped snapshot with keepPeriod = 3 and numberToKeep = 0
```

Comment:
below part should be dropped from this test case as it is seems a part of  `testRemoveTimestampedSnapshotsWithPreviousDate`